src/main/java/org/cyclops/evilcraftcompat/modcompat/forestry/ForestryModCompat.java

```diff
@@ -28,10 +29,10 @@ public String getModID() {
     @Override
     public void onInit(Step step) {
     	if(step == Step.INIT) {
+
 	        // Register the Undead Sapling.
-	        if(Configs.isEnabled(UndeadSaplingConfig.class)) {
-	            FMLInterModComms.sendMessage(getModID(), "add-farmable-sapling",
-						"farmArboreal@" + Block.REGISTRY.getNameForObject(UndeadSaplingConfig._instance.getBlockInstance()).toString() + ".0");
+	        if(Configs.isEnabled(UndeadSaplingConfig.class) && Configs.isEnabled(HardenedBloodShardConfig.class)) {
+                ForestryAPI.farmRegistry.registerFarmables("farmArboreal", new FarmableUndeadSapling());
```

Comment:
This will unfortunately cause a crash when Forestry is not installed.

You'll have to move this `ForestryAPI` call to a separate class. Have a look at how this is done with other mod compats, such as Immersive Engineering.